libcloud/common/base.py

```diff
@@ -495,6 +500,12 @@ def __init__(self, secure=True, host=None, port=None, url=None,
         if timeout:
             self.timeout = timeout
 
+        if retry_delay:
+            self.retry_delay = retry_delay
```

Comment:
I think it's better if we always set the instance variable - if user doesn't provide a value it will simply be `None`.